Advanced/Play Asset Delivery/Assets/PlayAssetDelivery/Runtime/ButtonInteraction.cs

```diff
@@ -16,7 +16,7 @@ public class ButtonInteraction : MonoBehaviour
         public void OnButtonClicked()
         {
             if (!AddressablesInitSingleton.Instance.HasInitialized)
-                Debug.LogError("Not finished initializing.");
+                Debug.LogError($"Cannot load object. AddressablesInitSingleton is still initializing.");
```

Comment:
I don't know if you want to mess with this but if it's not initialized and we had access to the internal init op we could call WaitForCompletion.  Not really a request as I'm good with this log as well, just a thought

Reply:
I like this. I'll try to convert this class to an AsyncOperation